### PR TITLE
Add ceph tiering metrics

### PR DESCRIPTION
**Pull request by ledgr**

### CODE REVIEW

File: ceph_exporter.py [hunk 1]
```diff
@@ -789,13 +843,36 @@ def start(self):
...
-parser.add_argument('--keyring', default='/etc/ceph/ceph.client.admin.keyring', help="Name of the file that contains key for authentication with the Ceph cluster, default is `/etc/ceph/ceph.client.admin.keyring`")
-parser.add_argument('--endpoint', default='tcp:9999', help="Twisted server endpoint specifier, default is `tcp:9999`")
-parser.add_argument('--executable', default='/usr/bin/ceph', help="Path to the Ceph command line client executable, default is `/usr/bin/ceph`")
+# ceph pg dump --format json
+# pg_stats_sum['num_flush'] => ceph_tier{fsid="$fsid",scope="cluster",type="num_flush")
+# pg_stats_sum['num_evict'] => ceph_tier{fsid="$fsid",scope="cluster",type="num_evict")
+# pg_stats_sum['num_promote'] => ceph_tier{fsid="$fsid",scope="cluster",type="num_promote")
+# ['pool_stats'][n]['stat_sum']['num_flush'] => ceph_tier{fsid="$fsid",scope="pool",pool="$pool_id",type="num_flush")
+# ['pool_stats'][n]['stat_sum']['num_evict'] => ceph_tier{fsid="$fsid",scope="pool",pool="$pool_id",type="num_evict")
+# ['pool_stats'][n]['stat_sum']['num_promote'] => ceph_tier{fsid="$fsid",scope="pool",pool="$pool_id",type="num_promote")
```
Comment (aurimasl) [inline ceph_exporter.py]: Why commented code is left here?

**Comment by aurimasl** [comment]
IMHO coding style changes should be in separate commit, it would be a lot easier to review changes.

**Comment by ledgr** [comment]
Well it should, but it was late and I didn't feel like doing one more PR ;)

File: ceph_exporter.py (hunk 1 above)
Comment (ledgr) [inline ceph_exporter.py]: To make it easier to understand what data correspond to what metrics. Otherwise, later it gets really hard to follow (for me personally)

Comment (aurimasl) [inline ceph_exporter.py]: It gets harder to read code with a lot of comments. Better put it in docs/wiki, or at least comments should have good style to make it easy to read. Now you have to "read and parse the code", which is not a code, just a doc.
